### PR TITLE
update continuous integration testing notes

### PR DESCRIPTION
**Pull request by tommadams**

### CODE REVIEW

File: testing/README.md [hunk 1]
```diff
@@ -12,21 +12,52 @@ Some UIs to check out:
 Testgrid (Test Results Dashboard): https://k8s-testgrid.appspot.com/sig-big-data
 Prow (Test-runner dashboard): https://prow.k8s.io/?repo=tensorflow%2Fminigo
 
-## Local testing
+## Updating continuous integration tests
 
-To test out changes to the docker image, first build the test-harness image:
+Build and push the test-harness image:
```
Comment (amj) [inline testing/README.md]: when is this necessary?  What part of our TF build etc. would need to change to merit rebuilding the image?

File: testing/README.md [hunk 2]
```diff
@@ -12,21 +12,52 @@ Some UIs to check out:
...
+
+You can test a PR sent to github locally using the following steps. This
+enables you to poke around if something's failing.
+
+```shell
+export PATH=$HOME/go/bin:$PATH
+cd $HOME  # Or somewhere else outside the minigo repo
+git clone git@github.com:kubernetes/test-infra.git
+cd test-infra/config
+./pj-on-kind.sh pull-tf-minigo-cc
```
Comment (amj) [inline testing/README.md]: @cjwagner this is it, you can't ever change the name of your script now ;)

Comment (cjwagner) [reply testing/README.md]: 🙃 
/shrug

File: testing/README.md (hunk 1 above)
Comment (tommadams) [reply testing/README.md]: I just had to do it to install `clang-format` on the testing image.

**Pull request merged**